torch/csrc/jit/passes/utils/subgraph_utils.h

```diff
@@ -46,6 +46,10 @@ TORCH_API void mergeNodeIntoSubgraphAndUpdateAliasing(
     Node* subgraphNode,
     AliasDb& db);
 
+TORCH_API std::vector<Node*> unmergeAliasedOutputs(
```

Comment:
if we're defining the api in the header here, can we move the rest of the impl to the subgraph_utils.cpp file ?